libc/src/math/nvptx/fmaximumf.cpp

```diff
@@ -0,0 +1,22 @@
+//===-- Implementation of the fmaximumf function for GPU
+//----------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include "src/math/fmaximumf.h"
+
+#include "src/__support/CPP/bit.h"
+#include "src/__support/common.h"
+#include "src/__support/macros/optimization.h"
+
+namespace LIBC_NAMESPACE {
+
+LLVM_LIBC_FUNCTION(float, fmaximumf, (float x, float y)) {
+  return __builtin_fmaximumf(x, y);
```

Comment:
Let remove all the specializations in `src/math/nvptx` for now.

Reply:
Ok. thanks